README.md

```diff
@@ -1,10 +1,11 @@
 # .github
+
 This repository exists to provide community health files for the `unicode-org` github organization.
 
-### Copyright & Licenses
+## Copyright & Licenses
 
-Copyright © 2020-2024 Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the United States and other countries.
+Copyright © 2020-2025 Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the United States and other countries.
 
-A CLA is required to contribute to this project - please refer to the [CONTRIBUTING.md](./.github/CONTRIBUTING.md) file (or start a Pull Request) for more information.
+To contribute to a Unicode Consortium repository, a Contributor License Agreement (CLA) is required. See [CONTRIBUTING.md](./.github/CONTRIBUTING.md) for more information.
```

Comment:
CONTRIBUTING.md will either be:
- a link to the org-wide CONTRIBUTING.md or
- a link to the in-repo `./CONTRIBUTING.md` (if there's a custom one)

If it is a link to the custom CONTRIBUTING.md, then that CONTRIBUTING.md will incorporate the `## Contributor License Agreement` section of the org-wide CONTRIBUTING.md.